app/helpers/record_helper.rb

```diff
@@ -62,6 +62,17 @@ def check_online?
     )
   end
 
+  # domains we consider relevant when evaluating links
+  def relevant_links
```

Comment:
Does this need to include walter.mit.edu? (The meta-question here is, "will this function behave the same way on testing servers as it does on prod?")

Reply:
We should be using production servers for everything, including development / testing. I'm hesitant to put test server support into our production code. We can chat further on Slack if I'm missing an important use case you are anticipating.